build_lib.py

```
@@ -594,6 +594,21 @@ def main(argv: list[str] | None = None) -> int:
             )
             if result.returncode != 0:
                 print(f"Warning: Failed to clear kernel cache (exit code {result.returncode})")
+
+            # Flush build output before printing diagnostics so log ordering is correct
+            sys.stdout.flush()
+            sys.stderr.flush()
+
+            # Print build diagnostics (subprocess ensures fresh import of rebuilt libraries)
+            result = subprocess.run(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)